compass/ocean/tests/dam_break/viz/__init__.py

```diff
@@ -0,0 +1,137 @@
+import xarray
+import numpy
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy import spatial
+from PIL import Image
+from collections import OrderedDict
+
+from compass.step import Step
+
+
+class Viz(Step):
+    """
+    A step for visualizing dam break results, as well as comparison with
+    experimental data and ROMS simulation output (Warner et al. 2013).
+
+    Attributes
+    ----------
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='viz')
+
+        self.add_input_file(filename='output.nc',
+                            target='../forward/output.nc')
+        filename = 'stationCoords.csv'
+        self.add_input_file(filename=filename, target=filename,
+                            database='dam_break')
+
+        filename = 'dam_break.png'
+        self.add_input_file(filename=filename, target=filename,
+                            database='dam_break')
+
+        observed_inputs = ['Station0.csv', 'Station4.csv', 'Station-5A.csv',
+                           'Station8A.csv', 'StationC.csv']
+        roms_inputs = ['0-sim.csv', '4-sim.csv', '-5A-sim.csv', '8A-sim.csv',
+                       'C-sim.csv']
+
+        for filename in observed_inputs:
+            self.add_input_file(filename=filename, target=filename,
+                                database='dam_break')
+        for filename in roms_inputs:
+            self.add_input_file(filename=filename, target=filename,
+                                database='dam_break')
+
+        self.add_output_file(filename='dam_break_comparison.png')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        # read output.nc
+        data = xarray.open_dataset('output.nc')
+        x_cell = data.xCell.values
+        y_cell = data.yCell.values
+        ssh = data.ssh.values
+        ssh = ssh + 0.6
```

Comment:
@cbegeman, What is the reason for the 0.6 offset here?

Reply:
That's the value of `config_dam_break_eta0`, which `bottomDepth` is set to. I added this commit [232e370](https://github.com/MPAS-Dev/compass/pull/417/commits/232e370a43cb2cb5f4a87b729d0998e0c66d3611) to use `bottomDepth` directly. Let me know if you like that solution.

Reply:
I like that solution very much. Thanks!